myStyles.css

```diff
@@ -218,11 +289,520 @@
+  margin-left: 10px;
+  margin-right: 10px;
+  font-family: 'Inter', sans-serif;
+}
+
+.workButton {
+  background-color: #ff6b00;
+  color: #fff;
+  padding-top: 3.5%;
+  padding-bottom: 3.5%;
+  padding-left: 3.5%;
+  padding-right: 3.5%;
+  width: fit-content;
+  text-align: left;
+  margin-left: 9%;
+  margin-bottom: 20%;
+  font-family: 'Inter', sans-serif;
+}
+
+.workButton:hover {
+  background-color: #eac2a6;
+  color: #ff6b00;
+  border-color: #ff6b00;
+}
+
+.workButton:active {
+  background-color: #ba5d1a;
+  color: #fff;
+  border-color: #ff6b00;
+}
+
+.workButton:disabled {
+  background-color: #c4b9b1;
+  color: #000;
+  border-color: #c4b9b1;
+}
+
+.worksCards {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 0.0125%;
+  text-align: center;
+  height: auto;
+}
+
+.card {
+  background-image: url("images/Work_Init/WorkCard.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.card h2 {
+  font-family: 'Crete Round', sans-serif;
+  text-align: left;
+  margin-left: 5%;
+  margin-right: 15%;
+  margin-top: 22%;
+  font-weight: normal;
+  padding-bottom: 6%;
+}
+
+.flexCardTitle {
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.card h3 {
+  font-family: 'Inter', sans-serif;
+  display: inline-block;
+  margin-left: auto;
+  padding-bottom: 5%;
+}
+
+.card h4 {
+  font-family: 'Inter', sans-serif;
+  display: inline-block;
+  margin-right: auto;
+  padding-bottom: 5%;
+}
+
+.card p {
+  font-family: 'Inter', sans-serif;
+  text-align: left;
+  margin-left: 5%;
+  margin-right: 5%;
+  padding-bottom: 4%;
+  display: block;
+}
+
+.worksListCard {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  align-items: center;
+  text-decoration: none;
+}
+
+.worksListCard li {
+  display: inline-block;
+  padding: 10px 15px;
+  text-decoration: none;
+  border: solid 1px;
+  margin-left: 10px;
+  margin-right: 10px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+}
+
+.worksListCard ul {
+  margin: auto;
+  padding-top: 6%;
+  padding-bottom: 15%;
+}
+
+.workButtonCard {
+  background-color: #ff6b00;
+  color: #fff;
+  padding-top: 6%;
+  padding-bottom: 3%;
+  padding-left: 4%;
+  padding-right: 4%;
+  text-align: center;
+  margin-top: auto;
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+.workButtonCard:hover {
+  background-color: #eac2a6;
+  color: #ff6b00;
+  border-color: #ff6b00;
+}
+
+.workButtonCard:active {
+  background-color: #ba5d1a;
+  color: #fff;
+  border-color: #ff6b00;
+}
+
+.workButtonCard:disabled {
+  background-color: #c4b9b1;
+  color: #000;
+  border-color: #c4b9b1;
+}
+
+.aboutMe {
+  background-color: black;
+  background-image: url("images/AboutMe/Background.png");
+  background-repeat: no-repeat;
+  background-size: 40%;
+  background-position: 100% 95%;
+}
```

Comment:
- [ ] Great work building your About Me section. 💯 However, please kindly ensure to pay special attention to `font-family` in the header, `background-color`, and the `margin-top` properties.🤓 Currently, your design still has some significant differences from the Figma design. 🔧

Figma design. 👇🏽
![f](https://github.com/alexansaa/Personal_Portafolio/assets/116701485/2dde965b-894a-499d-a667-a55bffb4e27c)


Your design. 👇🏽
![d](https://github.com/alexansaa/Personal_Portafolio/assets/116701485/1b1e490f-4b2b-4b81-9063-18a4b58ed9d9)